### DISCUSSION TITLE
Sensor "working" eines Rolladenaktors hinzufügen

### DISCUSSION THREAD

**Discussion by muellerjm** [SukramJ/aiohomematic#1824] [Q&A] [answered]
hallo,

ich bin von iob auf HA umgestiegen. Es gab in IOB  eine Automation die den Parameter "working" in channel 2 nutzte. In HA gibts den leider nicht. Kann ich das manuell über yaml als Sensor  integrieren? Vielen Danke

**Comment by SukramJ** [comment, marked as answer]
Steht in der [Anleitung](https://github.com/danielperna84/custom_homematic#unignore-device-parameters).

**Comment by muellerjm** [comment]
Vielen dank!
